ansibullbot/triagers/ansible.py

```diff
@@ -534,14 +534,17 @@ def create_actions(self, iw, actions, valid_labels):
                 return
 
         # TRIAGE!!!
-        if 'needs_triage' not in iw.labels and not iw.history.was_unlabeled('needs_triage') and not iw.history.was_unlabeled('triage'):
-            actions.newlabel.append('needs_triage')
-
-        # legacy "triage" -> "needs_triage"
-        if 'triage' in iw.labels:
-            if 'needs_triage' not in iw.labels:
+        # Determine triage labels when using the ansible/ansible repo
+        # TODO: Make this generic since needs_triage is used elsewhere (i.e, content team collections)
+        if iw.repo_full_name == "ansible/ansible":
```

Comment:
So I wonder if blocks guarded by `if iw.repo_full_name == "ansible/ansible"` (or similar) are good candidates to be extracted into a method that could be overridden in a custom triager:

```python
class CommunityTriager(AnsibleTriage):
    def triage(self):
        pass
```

Reply:
Probably. I tried to limit the scope of the changes before throwing myself into full blown refactoring :)